### PR TITLE
Added kevq.uk

### PR DESCRIPTION
**Pull request by kevquirk**
Added my site as I think I may just be weird enough to be included. I write about tech mostly, but also privacy and fish-keeping occasionally too. Let's be honest, fish are pretty weird!

I think most people who have their own personal site have some kind of weirdness to their personality. It's par for the course if want to put yourself online.

Oh, did I mention my avatar has no eyes?

Disclaimer: I do actually have eyes...

Hope I can be added, but if not, no hard feelings. 😊

### CODE REVIEW

**Comment by kevquirk** [comment]
Haven't heard back on this, so have removed the links from my footer. I assume that means you think my blog isn't appropriate for inclusion. That's totally cool. If you change your mind, I'll re-add them. :-)

All the best.

File: sites.js [hunk 1]
```diff
@@ -115,4 +115,9 @@ let sites = [
         url: 'https://gregoryhammond.ca',
         because: "Canadian geek who is trying to be unbiased, yet still talk about tech."
     }
```
Comment (jackmcdade) [inline sites.js:117]: Needs a comma after the brace. `},`

**Review by jackmcdade — CHANGES_REQUESTED** [review]
Needs a comma after that brace. Otherwise, cool site!

**Comment by kevquirk** [comment]
Will re-submit.

**Pull request closed without merging**